domain/src/Nft/NftId.php

```diff
@@ -5,7 +5,14 @@
 namespace Domain\Nft;
 
 use Domain\AggregateRootId;
+use Ramsey\Uuid\Uuid;
 
 final class NftId extends AggregateRootId
 {
+    private const NAMESPACE = '64d4faf2-d67a-4716-9b3c-bed1ca053068';
+
+    public static function fromNftId(string $nftId): static
+    {
+        return self::fromString(Uuid::uuid5(self::NAMESPACE, $nftId)->toString());
+    }
```

Comment:
ℹ️ This allows for the generation of a UUID which will always be the same, based on the NFT's ID. The same principle has been applied to `TaxYearId`